framework/context/reconciliationcanceledcontext/reconciliation_canceled_context.go

```diff
@@ -0,0 +1,61 @@
+// Package reconciliationcanceledcontext stores and accesses the canceled in
+// context.Context.
+package reconciliationcanceledcontext
```

Comment:
This package is what `canceledcontext` was before. So `reconciliationcanceledcontext` is for canceling the complete reconciliation of the object. I am not sure about the naming. Naming is hard. Feel free to suggest something better.

Reply:
What is `canceledcontext` now?

Reply:
> This package is what `canceledcontext` was before.

There is no `canceledcontext` anymore. It is now `reconciliationcanceledcontext`.

Reply:
I understand from the comment and code below.